### PR TITLE
Upgrade web forms v0.2

### PR DESCRIPTION
**Pull request by sadiqkhoja**
- Upgraded `getodk/web-forms` to v0.2 which brings in a appearances for Select questions.
- Added a CSS fix for the stopping bootstrap messing up with the styles of new web form component. We could have added this fix in the code of `web-forms` but it was decided that for now it is best to add in Central and update the docs of `web-forms` to let other host applications to do that same.

#### What has been done to verify that this works as intended?
Manually tested it to ensure various select appearances are rendered correctly.

#### Why is this the best possible solution? Were any other approaches considered?

Second commit in this PR reverts all the styling added by bootstrap and other css files for new Web Forms component. The approach felt cleanest to me. Other options were to dynamically removed the `<style>` tags when rending new Web Forms or when `meta.standalone` is true - that would also removed styles for dialog box that we show on "Send" button.

#### How does this change affect users? Describe intentional changes to behavior and behavior that could have accidentally been affected by code changes. In other words, what are the regression risks?

None

#### Does this change require updates to user documentation? If so, please file an issue [here](https://github.com/getodk/docs/issues/new) and include the link below.

Not sure we want to update the documentation for this feature or just announce it on the Forum.

#### Before submitting this PR, please make sure you have:

- [x] run `npm run test` and `npm run lint` and confirmed all checks still pass OR confirm CircleCI build passes
- [x] verified that any code or assets from external sources are properly credited in comments or that everything is internally sourced

### CODE REVIEW

File: src/assets/css/namespaces.css [hunk 1]
```diff
@@ -0,0 +1 @@
+@namespace svg "http://www.w3.org/2000/svg";
```
Comment (matthew-white) [inline src/assets/css/namespaces.css:1]: Why is this needed, out of curiosity? For `:not(svg)` to work below?

File: src/assets/scss/app.scss [hunk 2]
```diff
@@ -899,3 +899,11 @@ becomes more complicated.
   transition: opacity 0.6s 0.15s;
 }
 [data-mark-rows-deleted="false"] { display: none; }
+
+
+////////////////////////////////////////////////////////////////////////////////
+// Don't let styles defined in Central bleed into new Web Forms Component
+
+.odk-form :not(svg|*) {
```
Comment (matthew-white) [inline src/assets/scss/app.scss:907]: I feel like it'd be better to move this to `FormPreview`, since that's where all the other Web Form setup lives. `FormPreview` already houses some other Web Form-related CSS.

File: src/assets/css/namespaces.css (hunk 1 above)
Comment (sadiqkhoja) [reply src/assets/css/namespaces.css:1]: That's right, with `:not(svg|*)` I am excluding all elements of svg namespace. Without this exclusion, svg icons stop working.

File: src/assets/scss/app.scss (hunk 2 above)
Comment (sadiqkhoja) [reply src/assets/scss/app.scss:907]: I tried that but doesn't work because styles defined in the SFC are loaded after style coming from `web-form` component, in that case this rule reverts the `web-form` styles as well.

Comment (matthew-white) [reply src/assets/scss/app.scss:907]: Ah very interesting, that makes sense. :+1:
